+precice/@SolverInterface/private/preciceGateway.cpp

```diff
@@ -45,6 +45,9 @@ enum class FunctionID {
     setMeshTriangleWithEdges = 51,
     setMeshQuad = 52,
     setMeshQuadWithEdges = 53,
+    setMeshAccessRegion = 55,
+    getMeshVerticesAndIDs = 56,
+    isMeshConnectivityRequired = 54,
```

Comment:
Why not `54, 55, 56`? (Probably does not matter)

Reply:
Mixing up during merging both pull requests is definitely possible. In fact the order is not even incorrect, just the function name `isMeshConnectivityRequired` moved below the direct mesh access functions somehow. I will correct this.